content/news/2023.pml

```diff
@@ -54,6 +54,19 @@ Older News: <%=reflist%>
 </div>
 -->
 
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="microkit">
+  <a href="#microkit"><span class="tag-span"># </span></a>
+  25 October 2023: seL4 Microkit officially supported by the seL4 Foundation
+</div>
+<div>
+  <p>
+     The seL4 Microkit, formerly known as the Core Platform, is an operating system framework on top of seL4 provides a small set of simple abstractions that ease the design and implementation of statically structured systems on seL4, while still leveraging the kernel’s benefits of security and performance. The Microkit is distributed as an SDK that integrates with the developer’s build system of choice, significantly reducing the barrier to entry for new users of seL4.
+  </p>
```

Comment:
Do we want some extra context such as "The seL4 Microkit has been developed in collaboration between Breakaway Consulting and Trustworthy Systems and is now an official seL4 Foundation project, making it part of the seL4 eco-system."?

Reply:
@Ivan-Velickovic That's a good idea. I suggest adding the microkit logo (when done), TS logo and Breakaway logo too. I've pushed the news item date to 27 Oct.